ui/lib/kv/addon/components/page/secret/paths.hbs

```diff
@@ -16,15 +16,8 @@
 <div class="box is-fullwidth is-sideless is-paddingless is-marginless">
   {{#each this.paths as |path|}}
     <InfoTableRow @label={{path.label}} @labelWidth="is-one-third" @helperText={{path.text}} @truncateValue={{true}}>
-      {{! replace with Hds::Copy::Snippet }}
-      <CopyButton
-        class="button is-compact is-transparent level-right"
-        @clipboardText={{path.snippet}}
-        @buttonType="button"
-        @success={{fn (set-flash-message (concat path.label " copied!"))}}
-      >
-        <Icon @name="clipboard-copy" aria-label="Copy" />
-      </CopyButton>
+      {{! HDS Adoption: Replace with Hds::Copy::Snippet }}
```

Comment:
this is the only snippet I didn't replace because it was going to take more CSS finagling that it seems like it would 🙃

Reply:
Good question! Because it's in info table row - but I just had an idea that may be a quick fix - going to give it a shot

Reply:
Okay, I actually just think it doesnt look good with the copy button to the right 😅 
<img width="1159" alt="Screenshot 2023-09-15 at 12 45 09 PM" src="https://github.com/hashicorp/vault/assets/68122737/3ded28de-beb5-468c-b199-4ef0c3a1eef3">